observability/BUILD.bazel

```diff
@@ -0,0 +1,26 @@
+load("//:java_grpc_library.bzl", "java_grpc_library")
```

Comment:
We don't need bazel builds. Some of the projects in grpc-java have it because of legacy but new projects don't. You can reduce the overhead of maintaining bazel build files by not having this.

Reply:
> build rules to compile proto in build.gradle

Yes, that's a requirement even if you keep the bazel file around